polaris-react/src/components/ActionList/components/Item/Item.tsx

```diff
@@ -89,7 +97,7 @@ export function Item({
   const textMarkup = <span className={styles.Text}>{contentMarkup}</span>;
 
   const contentElement = (
-    <HorizontalStack blockAlign="center" gap="4">
+    <HorizontalStack blockAlign="center" gap="4" wrap={!truncate}>
```

Comment:
There could be use cases outside truncation where we want `ActionList` to not wrap. @emma-boardman's [PR](https://github.com/Shopify/polaris/pull/9029/files) proposes a `prop` to customize this. We'll still want wrap to be `false` for truncation on top of that prop.